cmd/ndm_daemonset/probe/uuid.go

```diff
@@ -53,6 +54,14 @@ func generateUUID(bd blockdevice.BlockDevice) (string, bool) {
 	// the partition and the unique data will be stored in sectors where consumers do not have access.
 
 	switch {
+	/*case bd.DeviceAttributes.DeviceType == blockdevice.BlockDeviceTypeLoop:
+	klog.Infof("device(%s) is a loop device, using path and node name")*/
+	case isDM(bd.DevPath):
+		// is a DM device, use the DM uuid
+		klog.Infof("device(%s) is a dm device, using DM UUID: %s", bd.DevPath, bd.DMInfo.DMUUID)
+		// TODO add a check if DM uuid is present, else what to do???
+		uuidField = bd.DMInfo.DMUUID
```

Comment:
can you add udevadm o/p showing the UUID for DM devices(virtual and physical both) in the PR?

Reply:
Done.